mathics/builtin/files_io/importexport.py

```diff
@@ -1372,11 +1375,17 @@ def eval(self, filename, evaluation, options={}):
 
     def eval_element(self, filename, element: String, evaluation, options={}):
         "Import[filename_, element_String, OptionsPattern[]]"
-        return self.eval_elements(
-            filename, ListExpression(element), evaluation, options
-        )
+        if element.get_string_value() in IMPORTERS.keys():
```

Comment:
maybe it would be a good idea to move the implementation of these methods to `mathics.eval.files_io`

Reply:
Thank you for the comments, @mmatera. Would it make sense to move all related (to import/export) `eval_xxx` functions there?

First, I will implement the 2nd point above, and then, soon after receiving your response, will move the implementation to `mathics.eval.files_io`.

Reply:
Thank you for the explanation and the context, @rocky. Indeed, this helps. Since the changes for this PR are small, I'll add them to the existing methods in `mathics.builtins`. From the next PR on, I'll implement substantial code as helper functions in `mathics.eval`.